sdk/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@devcycle/devcycle-react-sdk",
-  "version": "1.2.22-alpha.0",
+  "version": "1.2.22",
```

Comment:
do not change, this can stay as is with the alpha tag.. otherwise this will publish a new version of the React SDK with version `1.2.22`